tests/performance/load-tests/load-test.sh

```diff
@@ -107,8 +110,10 @@ function runTest() {
       succeeded=$((succeeded + 1))
     else
       print_error "Timeout waiting for dw$i to become ready or an error occurred."
+      ws_name=$(kubectl get dw dw$i --template='{{.status.devworkspaceId}}')
       kubectl describe dw dw$i >logs/dw$i-log.log
-      kubectl logs $(kubectl get dw dw$i --template='{{.status.devworkspaceId}}') >logs/dw$i-pod.log || true
+      cat logs/events.log | grep $ws_name >logs/dw$i-events.log
```

Comment:
Why not to use devworkspace id to name log file instead of $i?
It will simplify location of problem workspace.

Reply:
dw$i is a name of created devworkspace. That will be not easy to find workspace log for devworkspace.

Reply:
I see.
Then it would improve the readiness of script if you declare `ws_name="dw$i"` explicitly, IMHO.